tasks/Functions-client/request.js

```diff
@@ -90,28 +91,36 @@ task("functions-request", "Initiates a request from an Functions client contract
     const linkBalance = subInfo[0]
     if (subInfo[0].lt(estimatedCostJuels)) {
       throw Error(
-        `Subscription ${subscriptionId} does not have sufficient funds. The estimated cost is ${estimatedCostJuels} Juels LINK, but has balance of ${linkBalance}`
+        `Subscription ${subscriptionId} does not have sufficient funds. The estimated cost is ${estimatedCostJuels} Juels LINK, but the subscription only has a balance of ${linkBalance} Juels LINK`
```

Comment:
We need to divide it by 10^18 and show a decimal LINK, otherwise the numbers are too big to eyeball